.gitignore

```diff
@@ -50,6 +50,7 @@ profile-*
 
 # vscode
 .vscode
+*code-workspace
```

Comment:
FYI: this is probably going to cause a conflict in your other PR

Reply:
yes, I will remove it.